Orange/widgets/visualize/owmosaic.py

```diff
@@ -295,9 +298,8 @@ def initCombos(self, data):
         self.attr4Combo.addItem("(None)")
 
         for attr in chain(data.domain, data.domain.metas):
-            if attr.is_discrete:
-                for combo in [self.attr1Combo, self.attr2Combo, self.attr3Combo, self.attr4Combo]:
-                    combo.addItem(self.icons[attr], attr.name)
+            for combo in [self.attr1Combo, self.attr2Combo, self.attr3Combo, self.attr4Combo]:
```

Comment:
Meta attributes can also be of other types, like strings or time stamps. You should only add numeric and discrete (`is_discrete` and `is_continuous`) here.